README.md

```diff
@@ -79,6 +79,19 @@ It starts etcd and simulator-server locally.
 
 To run the frontend, please see [README.md](./web/README.md) on ./web dir.
 
+## Existing cluster importing
```

Comment:
Let's specify this feature as Beta for now. I think we need to improve this more to mark this feature as Stable. #142


```suggestion
## [Beta] Existing cluster importing
```